### ISSUE TITLE
NOT masked secret parameters in "liveaction"

### ISSUE THREAD

**Issue by maxim-s-barabash** [StackStorm/st2#3380]
st2 version 2.2.1.

This applies to logs and st2api.

/st2/v1/executions/id

```
{
parameters": {
    "user_password": "********", 
},
"liveaction":
   "parameters":  {
          "user_password": "password",  <--  secret: true
  }
}
```

**Comment by Kami** [comment]
Thanks for reporting this.

This issue has been fixed recently as part of https://github.com/StackStorm/st2/pull/3341 and the fix will be included in the next release (v2.3.0) which should be out some time in the next couple of weeks.

In addition to that, I would encourage you to review new "Secrets Masking" document which describes how it works and spells out the limitations of relying just on secrets masking - https://docs.stackstorm.com/latest/reference/secrets_masking.html.

**Comment by maxim-s-barabash** [comment]
Thanks!
